src/Simplex/Messaging/Agent/Store.hs

```diff
@@ -46,14 +46,20 @@ class Monad m => MonadAgentStore s m where
   setRcvQueueStatus :: s -> RcvQueue -> QueueStatus -> m ()
   setRcvQueueActive :: s -> RcvQueue -> VerificationKey -> m ()
   setSndQueueStatus :: s -> SndQueue -> QueueStatus -> m ()
+  updateSignatureKey :: s -> SndQueue -> SignatureKey -> m ()
+
+  -- Confirmations
+  createConfirmation :: s -> TVar ChaChaDRG -> NewConfirmation -> m ConfirmationId
+  getConfirmation :: s -> ConfirmationId -> m Confirmation
+  approveConfirmation :: s -> ConfirmationId -> ConnInfo -> m ()
+  getApprovedConfirmation :: s -> ConnId -> m Confirmation
+  removeApprovedConfirmation :: s -> ConnId -> m ()
```

Comment:
this doesn't seem used at all - probably if we want to remove confirmation after it's accepted we can just remove all of them?

Reply:
TODO remove all

Reply:
done